app/models/rss_log.rb

```diff
@@ -382,7 +382,7 @@ def detail
     elsif !target ||
           tag.to_s.match?(/^log_#{target_type}_(merged|destroyed)/)
       :rss_destroyed.t(type: target_type)
-    elsif !time
+    elsif !time || time < target.created_at + 1.minute
```

Comment:
We definitely want to check the logs for the activity log to make sure this isn't causing N extra queries.  (Anything else use RssLog#detail we care about?)

Reply:
I believe only the matrix box presenter calls RssLog#detail.

From inspecting my GraphQL/GraphiQL queries, I realized even line 382 above (`elsif !target`) queries the target. Neither seem to affect the presenter though, due to eager loading, and this additional check in line 385 doesn't seem to cause any further queries.

It seems the eager loading needs to happen via the use of some gems in my GraphQL setup. (It is recursively re-querying the same observations when it goes to get their rss_log.detail!)